stories/components/Error.stories.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react'
+import Error from '../../components/Error'
+import { MockedProvider } from '@apollo/react-testing'
+
+export default {
+  components: Error,
+  title: 'Components/Error'
+}
+
+export const NotFound: React.FC = () => {
+  return (
+    <MockedProvider addTypename={false}>
```

Comment:
the `mockedprovider` shouldnt be needed since the `Error` component doesn't depend on any graphql related things